nilearn/image/image.py

```diff
@@ -531,9 +532,9 @@ def mean_img(imgs, target_affine=None, target_shape=None, verbose=0, n_jobs=1):
 
     """
     imgs = stringify_path(imgs)
-    is_str = isinstance(imgs, str)
+    is_path = isinstance(imgs, str) or isinstance(imgs, pathlib.Path)
```

Comment:
`imgs` should be a string (or an iterable) at this point because `stringify_path` has been called on it:

https://github.com/nilearn/nilearn/blob/1b7994104d08a117b859e784f3de567be87f0e59/nilearn/_utils/helpers.py#L153-L168

Reply:
Sorry, I should not have changed `mean_img`. The error appears when calling `nilearn.image.new_image_like`. I added a new commit reversing the change.